### PR TITLE
Development

### PR DESCRIPTION
**Pull request by princemaxi**

### CODE REVIEW

File: resources/CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+ul {
+    display: flex;
+    align-items: center;
+    gap: 25px;
+}
+
+li {
+    list-style: none;
+
```
Comment (Phildeoner) [inline resources/CSS/style.css:47]: Remove this empty line

File: resources/CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+a {
+    text-decoration: none;
+    font-weight: 550;
+    color: rgb(104, 103, 103);
+}
+
+a:hover {
+    text-decoration: underline;
+    transition: 1s;
+}
```
Comment (Phildeoner) [inline resources/CSS/style.css:59]: Add an empty line after the closing curly brace

File: resources/CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    color: whitesmoke;
+}
+
+h1 {
+    font-size: 40px;
+    color: rgb(104, 103, 103);
+    text-shadow: silver 0.5px 0.5px 0.5px;
+    text-emphasis-color:inherit;
+    text-decoration:salmon;
+
```
Comment (Phildeoner) [inline resources/CSS/style.css:86]: Remove this empty line

File: resources/CSS/style.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+.gal_button {
+    padding: 15px 60px;
+    background-color: rgb(104, 103, 103);
+    color: whitesmoke;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+    margin: 50px;
+    font-size: 25px;
+
```
Comment (Phildeoner) [inline resources/CSS/style.css:98]: Remove this empty line

**Review by Phildeoner — COMMENTED** [review]
Hi @princemaxi 👋, Wow you did an amazing job well-done, consider the comments then proceed to merge your work.
Happy coding😊😊

File: resources/CSS/style.css (hunk 4 above)
Comment (princemaxi) [reply resources/CSS/style.css:98]: Done, thank you

**Pull request merged**